index.js

```diff
@@ -47,20 +47,7 @@ const types = {
     }
 };
 
-module.exports = (src, type) => {
-    if (!src || !type) {
-        throw new Error('`src` and `type` required');
-    }
-
-    const chosenType = types[type];
-    const $ = cheerio.load(src);
-
-    return $(chosenType.selector).map((i, el) => {
-        return $(el).attr(chosenType.attribute);
-    }).toArray();
-};
-
-module.exports.raw = (src, type) => {
+function oust(src, type, raw) {
     if (!src || !type) {
         throw new Error('`src` and `type` required');
     }
```

Comment:
Maybe add another check for `chosenType` so we don't end up with `Cannot read property 'attribute' of undefined`

Reply:
But `chosenType = types[type]`, the same error happens in master too :)

Feel free to push here and I will check out your solution.

Reply:
Doesn't mean we have to keep it that way ;)